### PR TITLE
handle detection list removal errors

### PR DESCRIPTION
**Pull request by timabrmsn**

### CODE REVIEW

File: src/code42cli/cmds/alert_rules.py [hunk 1]
```diff
@@ -68,7 +69,12 @@ def add_user(state, rule_id, username):
 @sdk_options()
 def remove_user(state, rule_id, username):
     """Remove a user from an alert rule."""
-    _remove_user(state.sdk, rule_id, username)
+    try:
+        _remove_user(state.sdk, rule_id, username)
+    except Py42BadRequestError:
```
Comment (unparalleled-js) [inline src/code42cli/cmds/alert_rules.py:74]: Is there a chance that the rule ID or username could somehow cause a bad request (from the CLI) for other reasons?

**Comment by unparalleled-js** [comment]
Should mention this in the Changelog

File: src/code42cli/cmds/departing_employee.py [hunk 2]
```diff
@@ -46,7 +47,14 @@ def add(state, username, cloud_alias, departure_date, notes):
 @sdk_options()
 def remove(state, username):
     """Remove a user from the departing-employee detection list."""
-    _remove_departing_employee(state.sdk, username)
+    try:
+        _remove_departing_employee(state.sdk, username)
+    except Py42NotFoundError:
```
Comment (unparalleled-js) [inline src/code42cli/cmds/departing_employee.py:52]: in `err.response.text`, it says `Could not find item for user: <id>`... Probably want to leverage that in py42 at least.

**Comment by unparalleled-js** [comment]
Testing passed

File: src/code42cli/cmds/alert_rules.py (hunk 1 above)
Comment (timabrmsn) [reply src/code42cli/cmds/alert_rules.py:74]: I just tested these cases and they both provide correct error messages.

**Pull request merged**